apis/nucleus/src/viz.js

```diff
@@ -104,6 +104,12 @@ export default function viz({
         }
         return false;
       },
+      getPropertyPanelDefinition() {
+        if (mountedReference && successfulRender) {
+          return originalExtensionDef ? originalExtensionDef.ext : cellRef.current.getExtensionDefinition().definition;
```

Comment:
should it not be `originalExtensionDef.definition`?

Reply:
yes, right!